src/main/java/org/folio/bulkops/util/EntityPathResolver.java

```diff
@@ -8,19 +8,25 @@
 import org.folio.bulkops.domain.bean.Item;
 import org.folio.bulkops.domain.bean.User;
 import org.folio.bulkops.domain.dto.EntityType;
+import org.folio.spring.FolioExecutionContext;
+import org.folio.spring.FolioModuleMetadata;
+import org.folio.spring.scope.FolioExecutionContextSetter;
 import org.springframework.stereotype.Component;
 
 import static java.lang.String.format;
 import static org.folio.bulkops.domain.dto.IdentifierType.ID;
+import static org.folio.bulkops.util.FolioExecutionContextUtil.prepareContextForTenant;
 
 @Component
 @AllArgsConstructor
 public class EntityPathResolver {
 
   private HoldingsClient holdingsClient;
+  private final FolioModuleMetadata folioModuleMetadata;
+  private final FolioExecutionContext folioExecutionContext;
 
-  public String resolve(EntityType type, BulkOperationsEntity entity) {
-
+  public String resolve(EntityType type, BulkOperationsEntity original) {
```

Comment:
What was the purpose of renaming the variable entity to original? This utility method should be applied not only to the original entity but potentially to any entity whose type needs to be determined.

Reply:
From one side for item in ecs environment for inventory link resolving it is necessary to know tenant,  the tenant is populated in original json files. The other side previous code is used this method for original entities only.

Reply:
Updated